test/integration/lnd-utils.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { MainBook } from "src/mongodb"
+import { baseLogger } from "src/logger"
+import { updateRoutingFees } from "src/lndUtils"
+import { revenueFeePath } from "src/ledger/ledger"
+import {
+  createInvoice,
+  getForwards,
+  lnd1,
+  lndOutside1,
+  lndOutside2,
+  pay,
+} from "test/helpers"
+
+afterAll(async () => {
+  jest.restoreAllMocks()
+})
+
+describe("lndUtils", () => {
+  describe("updateRoutingFees", () => {
+    it("sets routing fee correctly", async () => {
```

Comment:
what is the goal of this test?

Reply:
basically test updateRoutingFees method (previous test https://github.com/GaloyMoney/galoy/blob/main/test/integration/09f-routing.spec.ts and related to https://github.com/GaloyMoney/galoy/blob/main/test/integration/04b-lightning-payment.spec.ts#L559). The method basically test how "routing fee" mainbook entry is set and the logic regarding routingFeeLastEntry in metadata

Reply:
how do you ensure that there is no direct channel between lndOutside1 and lndOutside2? shouldn't the order of test for this matters?

Reply:
:) https://github.com/dolcalmi/galoy/blob/refactor_test_stg3a/test/integration/01-setup/01-lightning-channels.spec.ts#L43 that is the reason to use a new channel configuration and only setup channels in the setup script

Reply:
I'm not sure to full understand. 

https://github.com/dolcalmi/galoy/blob/a923ee88f2f084966ecac900521b2d95f2afe03a/test/integration/01-setup/01-lightning-channels.spec.ts

this file is run only once or for every tests?

Reply:
no, the channel setup only runs once for the integration test, what I did was to no setup the link between lndOutside1 and lndOutside2 (the channel closing was causing problems after finish the lightning sending tests) so all the payments between them go through lnd1